LayoutTests/webrtc/RTCDataChannelInit-maxPacketLifeTime-enforce-range.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>RTCDataChannelInit maxPacketLifeTime [EnforceRange] unsigned short tests</title>
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<script>
+'use strict';
+
+// Helper to create a basic RTCPeerConnection
+function createPC() {
+  return new RTCPeerConnection();
+}
+
+// Test valid values within unsigned short range (0-65535)
+test(() => {
+  const pc = createPC();
+  const channel = pc.createDataChannel('test', { maxPacketLifeTime: 0 });
+  assert_equals(channel.maxPacketLifeTime, 0);
+}, 'maxPacketLifeTime with value 0 should succeed');
+
+test(() => {
+  const pc = createPC();
+  const channel = pc.createDataChannel('test', { maxPacketLifeTime: 1 });
```

Comment:
Note that ideally the tests would be in WPT since the browsers are not aligned with the specification here and we want them to.